src/robot/output/pyloggingconf.py

```diff
@@ -22,6 +22,7 @@
 
 LEVELS = {'TRACE': logging.NOTSET,
           'DEBUG': logging.DEBUG,
+          'ERROR': logging.ERROR,
           'INFO': logging.INFO,
           'WARN': logging.WARNING}
```

Comment:
Would be better to have these in priority order. In other words, ERROR should be last.